src/factories/request.js

```diff
@@ -108,6 +110,11 @@ class RequestFactory {
           headers['Content-Type'] = 'application/json'
         }
 
+        if (isFormData && isNode()) {
+          // construct proper headers for FormData requests in Node.js. assumes a FormData object built with form-data lib
+          headers['Content-Type'] = body.getHeaders()['Content-Type']
```

Comment:
Since it's possible for someone to use `formdata-node`, which doesn't have the `getHeaders` method, I would add a check here. If the method doesn't exist then this call will fail so we may want to fail here instead

Reply:
@cjmarkham `isFormData` would actually fail in that case. so the user would probably end up with a failed request rather than a failed execution i think. which obvs isnt ideal either but im not sure we can cater for every eventuality? probably we want to encourage the happy path of just allowing the the SDK to handle anything formdata related